aksel.nav.no/website/sanity/schema/objects/shared/riktekst.tsx

```diff
@@ -65,10 +65,19 @@ export const block = {
         component: ({ children }) => <InlineCode>{children}</InlineCode>,
       },
       {
-        title: "KBD",
+        title: "Quote",
+        value: "quote",
+        icon: () => (
+          <span className="font-semibold" aria-label="quote">
```

Comment:
Looks like Sanity adds `aria-label="Quote"` to the button.
```suggestion
          <span className="font-semibold">
```

Reply:
Nice, this was a problem before where none of the buttons had any labels or titles, leading to them being inaccessible for screenreaders.